components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/main/java/org/wso2/carbon/identity/notification/sender/tenant/config/NotificationSenderManagementConstants.java

```diff
@@ -102,13 +102,15 @@ private NotificationSenderManagementConstants() {
     public static final String SMTP_PORT_PROPERTY = "mail.smtp.port";
     public static final String HTTP_URL_PROPERTY = "http.url";
     public static final String CONSTANT_HTTP_POST = "HttpPost";
+    public static final String CHANNEL_TYPE_PROPERTY = "channel.type";
 
     public static final List<String> INTERNAL_PROPERTIES =
             Collections.unmodifiableList(Arrays.asList(STREAM_NAME, STREAM_VERSION, PUBLISHER_TYPE_PROPERTY));
     public static final List<String> PROPERTIES_TO_SKIP_AT_ADAPTER_CONFIG =
             Collections.unmodifiableList(
                     Arrays.asList(STREAM_NAME, STREAM_VERSION, PUBLISHER_TYPE_PROPERTY, INLINE_BODY_PROPERTY));
 
+
```

Comment:
remove this additional new line

Reply:
resolved in https://github.com/wso2-extensions/identity-event-handler-notification/pull/179/commits/6feda46da33ad906dc43f6c786db85b7f0bce845